### ISSUE TITLE
tf.keras.BatchNormalization  Cannot be quantified

### ISSUE THREAD

**Issue by dlml** [tensorflow/tensorflow#22609]
Batch normalization resolution requires that mean, multiplier and offset arrays be constant.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by dlml** [comment]
i have solved this bug

**Comment by neney1234** [comment]
how?

**Comment by brucechou1983** [comment]
@dlml I got the same problem. How did you solve it?
